packages/core-p2p/src/peer-communicator.ts

```diff
@@ -225,12 +225,6 @@ export class PeerCommunicator implements P2P.IPeerCommunicator {
                 if (process.env.CORE_P2P_PEER_VERIFIER_DEBUG_EXTRA) {
                     this.logger.debug(`Response error (peer ${peer.ip}) : ${error.message}`);
                 }
-                break;
-            case SocketErrors.Timeout:
-            case "TimeoutError":
-            case "CoreSocketNotOpenError":
-                this.emitter.emit("internal.p2p.disconnectPeer", { peer });
```

Comment:
You cannot remove this line. The peer still needs to be removed properly.

Reply:
Same line will be executed a few lines below, in the `default:`: https://github.com/ArkEcosystem/core/pull/2731/files/240374fdf1d5166562e7875d282c0792c54e0445#diff-f9669fac2b5d6c80fe63216e2a805072R231

Reply:
Yeah, I misse

Reply:
Oh, right. But still, it was deliberately disabled for timeouts, because they clutter the log too much. At least wrap it in `CORE_P2P_PEER_VERIFIER_DEBUG_EXTRA`. Which btw. should be renamed by now, since it's universally used inside `core-p2p`.

Reply:
Wrapped it in `CORE_P2P_PEER_VERIFIER_DEBUG_EXTRA` in f57f88825.